### ISSUE TITLE
SqlRender handling of IF statements doesn't support nested conditions

### ISSUE THREAD

**Issue by chrisknoll** [OHDSI/SqlRender#31]
I'm trying to do the following:

``` R
testAnalysisToRun <- c(1500: 1505)
sqlTest <- "{ @runCostAnalysis & (1502 in (@list_of_analysis_ids) | 1503 in (@list_of_analysis_ids))} ? { Running Analysis } : { Not Running Analysis }"
renderSql(sqlTest, list_of_analysis_ids = testAnalysisToRun, runCostAnalysis = TRUE)$sql
# Returns: " Not Running Analysis ", should return "Running Analysis"
renderSql(sqlTest, list_of_analysis_ids = testAnalysisToRun, runCostAnalysis = FALSE)$sql
# Returns: " Not Running Analysis " (this is correct but because of above, probably not working right)
```

The idea is I'd like to do the following in an IF clause of sqpl Rrender:

If ( A AND (B OR C OR D)) 
then { do it} 

I'm trying to do it this way because of issue #30, where I'd prefer to do:

```
if (A) {
  If (B Or C)
  {
    -- Do case when B or C is true
  )

  if (D or E)
  (
    -- Do case when D or E 
  )
}
```

To rewrite this without nested:

```
if (A & (B or C) ) {
  -- Do case when B or C is true
}
~~






```

**Comment by schuemie** [comment]
Yep, it's a bug!

Should be fixed in 17771a9cf8e41c2833bfcc0dc63da650eeb9f139. Please test.

**Comment by chrisknoll** [comment]
Will do, excellent test cases in that commit :+1:
